### PR TITLE
Add document for int8 object detection quantization

### PR DESCRIPTION
**Pull request by lidanqing-intel**
add document for int8 object detection quantization.

Performance and accuracy are achieved on Intel(R) Xeon(R) Gold 6248 CPU

### CODE REVIEW

**Comment by lidanqing-vv** [comment]
Hi @wojtuss Please help reivew. Thanks !

**Comment by lidanqing-vv** [comment]
@bingyanghuang

**Comment by bingyanghuang** [comment]
@luotao1 @wzzju @juncaipeng I just found that there is one more readme needs to be merged. @Sand3r- Please help review this PR.

**Comment by lidanqing-vv** [comment]
Hi, @juncaipeng please paste the 6271 benchmark results here so that I could update the doc. Thank you very much!

**Comment by juncaipeng** [comment]
### Accuracy and Performance benchmark for Image Classification models

>**I. Top-1 Accuracy on Intel(R) Xeon(R) Gold 6271**

| Model        | FP32 Accuracy   | INT8 Accuracy   | Accuracy Diff(FP32-INT8)   |
| :----------: | :-------------: | :------------:  | :--------------:           |
| GoogleNet    |  70.50%         |  70.08%         |   0.42%                    |
| MobileNet-V1 |  70.78%         |  70.41%         |   0.37%                    |
| MobileNet-V2 |  71.90%         |  71.34%         |   0.56%                    |
| ResNet-101   |  77.50%         |  77.43%         |   0.07%                    |
| ResNet-50    |  76.63%         |  76.57%         |   0.06%                    |
| VGG16        |      72.08%       |  72.05%         |   0.03%                   |
| VGG19        |  72.57%         |  72.57%         |   0.00%                    |

>**II. Throughput on Intel(R) Xeon(R) Gold 6271 (batch size 1 on single core)**

| Model        | FP32 Throughput(images/s)  | INT8 Throughput(images/s) | Ratio(INT8/FP32)|
| :-----------:| :------------:             | :------------:            | :------------:  |
| GoogleNet    |    32.51                   |    63.23                  |   1.94          |
| MobileNet-V1 |    73.01                   |   191.59                  |   2.62          |
| MobileNet-V2 |    90.68                   |   154.14                  |   1.70          |
| ResNet-101   |     7.15                   |    24.64                  |   3.45          |
| ResNet-50    |    13.24                   |    45.42                  |   3.43          |
| VGG16        |         3.48               |          10.07            |       2.89     |
| VGG19        |     2.80                   |     8.64                  |   3.09          |

### Accuracy and Performance benchmark for Object Detection models

>**I. mAP on Intel(R) Xeon(R) Gold 6271 (batch size 1 on single core):**

| Model        | FP32 Accuracy   | INT8 Accuracy   | Accuracy Diff(FP32-INT8)   |
| :----------: | :-------------: | :------------:  | :--------------:           |
| Mobilenet-SSD| 73.80%         |  73.17%         |   0.63%                    |

>**II. Throughput on Intel(R) Xeon(R) Gold 6271 (batch size 1 on single core)**

| Model        | FP32 Throughput(images/s)  | INT8 Throughput(images/s) | Ratio(INT8/FP32)|
| :-----------:| :------------:             | :------------:            | :------------:  |
| Mobilenet-SSD    |    37.8180       | 115.0604 | 3.04 |

**Comment by luotao1** [comment]
@lidanqing-intel @bingyanghuang 请尽快更新下数据。`full_pascalvoc_test_preprocess.py`的修改会影响单测，需要过ci么？

**Comment by bingyanghuang** [comment]
> @lidanqing-intel @bingyanghuang 请尽快更新下数据。`full_pascalvoc_test_preprocess.py`的修改会影响单测，需要过ci么？

正在更新，full_***.py更改都是文档修改，我觉得可以不过。但是如果确实认为会造成影响，那就过一下CI。

File: paddle/fluid/inference/tests/api/full_pascalvoc_test_preprocess.py [hunk 1]
```diff
@@ -265,36 +265,54 @@ def run_convert():
...
+        "--mean_value",
+        type=str,
+        default=MEAN_VALUE,
+        help="Image preprocess with mean_value")
+    parser.add_argument(
+        "--ap_version",
+        type=str,
+        default=AP_VERSION,
+        help="Image preprocess with ap_version")
+    #parser.print_help()
```
Comment (luotao1) [inline paddle/fluid/inference/tests/api/full_pascalvoc_test_preprocess.py]: 这个可以去掉

File: paddle/fluid/inference/tests/api/full_pascalvoc_test_preprocess.py [hunk 2]
```diff
@@ -265,36 +265,54 @@ def run_convert():
...
-    )
+        description="Convert the full pascalvoc val set or local data to binary file.",
+        usage=None,
+        add_help=True)
     parser.add_argument(
         '--choice', choices=['local', 'VOC_test_2007'], required=True)
     parser.add_argument(
         "--data_dir",
-        default="/home/li/AIPG-Paddle/paddle/build/third_party/inference_demo/int8v2/pascalvoc_small",
+        default="./third_party/inference_demo/int8v2/pascalvoc_small",
```
Comment (luotao1) [inline paddle/fluid/inference/tests/api/full_pascalvoc_test_preprocess.py]: > full_***.py更改都是文档修改

default值变化也没有关系么

Comment (bingyanghuang) [reply paddle/fluid/inference/tests/api/full_pascalvoc_test_preprocess.py]: - 已更新benchmark信息，在等军才int8v2的验证信息，如果需要更改可在这个PR中一起更改合入，因为修改的是同一个文件。

- 已把python 修改移除，PR #20475 已提出过CI

**Pull request merged**